ui/admin/app/routes/scopes/scope/session-recordings.js

```diff
@@ -68,4 +70,28 @@ export default class ScopesScopeSessionRecordingsRoute extends Route {
       };
     }
   }
+
+  /**
+   * Reapplies storage policy dates to session recording
+   * @param {SessionRecordingModel}
+   */
+  @action
+  @notifyError(({ message }) => message, { catch: true })
+  @notifySuccess(function () {
+    const { end_time } = this.modelFor(
+      'scopes.scope.session-recordings.session-recording',
```

Comment:
I don't think we're meant to use `modelFor` to pull a descendant as it might cause undefined behavior.